### PR TITLE
Refactoring variables

### PR DESCRIPTION
**Pull request by jppgomes**
**Proposed changes**:
- Change some variables names and break method
- closes https://github.com/RasaHQ/rasa/issues/8463
- closes https://github.com/RasaHQ/rasa/issues/8527

### CODE REVIEW

**Comment by sara-tagger** [comment]
Thanks for submitting a pull request 🚀 @melindaloubser1 will take a look at it as soon as possible ✨

**Review by melindaloubser1 — CHANGES_REQUESTED** [review]
Thanks for contributing! Can you address the undefined name errors first?

**Comment by jppgomes** [comment]
@melindaloubser1  yes, of course i'll fix this

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 repos:
 -   repo: https://github.com/ambv/black
-    rev: 19.10b0
+    rev: 20.8b1
     hooks:
     - id: black
 -   repo: https://github.com/thlorenz/doctoc
-    rev: master
+    rev: v2.0.0
```
Comment (melindaloubser1) [inline .pre-commit-config.yaml:7]: what's the motivation for the version changes?

**Review by melindaloubser1 — COMMENTED** [review]
Besides the changes in `black` version, this looks good!

File: .pre-commit-config.yaml (hunk 1 above)
Comment (jppgomes) [reply .pre-commit-config.yaml]: sorry, it wasn't my intention to change this file, I undid the changes

**Review by melindaloubser1 — APPROVED** [review]

**Comment by TyDunn** [comment]
@melindaloubser1 Can we this merged in time for Rasa Open Source 2.6 tomorrow?

**Comment by melindaloubser1** [comment]
Sure I'll resolve the conflicts now

**Comment by melindaloubser1** [comment]
@jppgomes I'm not able to update your branch with the merge conflict resolution, can you resolve it please?

**Comment by jppgomes** [comment]
@melindaloubser1 yes!!

**Pull request merged**